requests/__init__.py

```diff
@@ -81,8 +82,9 @@ def check_compatibility(urllib3_version, chardet_version):
 
 # Attempt to enable urllib3's SNI support, if possible
 try:
-    from urllib3.contrib import pyopenssl
-    pyopenssl.inject_into_urllib3()
+    if not 'REQUESTS_DONT_USE_PYOPENSSL' in os.environ:
```

Comment:
Pedantic: I'd prefer `if x not in y` to `if not x in y`. 😉

Reply:
yeah flake8 always tells me the same thing :)

Reply:
Alright, you want me to change this and then merge?